<html>
<head>
<title>TestNG:  PersonalLoan</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>PersonalLoan</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>8/1/0</td>
</tr><tr>
<td>Started on:</td><td>Thu Sep 10 17:22:01 PDT 2020</td>
</tr>
<tr><td>Total time:</td><td>0 seconds (116 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='test.day1.lasTTest()'><b>lasTTest</b><br>Test class: test.day1</td>
<td><div><pre>java.lang.AssertionError: expected [true] but found [false]
	at test.day1.lasTTest(day1.java:18)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 20 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1897109193", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1897109193'><pre>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertTrue(Assert.java:44)
	at org.testng.Assert.assertTrue(Assert.java:54)
	at test.day1.lasTTest(day1.java:18)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
</pre></div></td>
<td>0</td>
<td>test.day1@242b836</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='test.day3.APIcarloan()'><b>APIcarloan</b><br>Test class: test.day3</td>
<td></td>
<td>0</td>
<td>test.day3@3f6f6701</td></tr>
<tr>
<td title='test.day3.AWeblogin()'><b>AWeblogin</b><br>Test class: test.day3</td>
<td></td>
<td>0</td>
<td>test.day3@3f6f6701</td></tr>
<tr>
<td title='test.day3.Mobilesignout()'><b>Mobilesignout</b><br>Test class: test.day3<br>Parameters: firstsetusername, password</td>
<td></td>
<td>0</td>
<td>test.day3@3f6f6701</td></tr>
<tr>
<td title='test.day3.Mobilesignout()'><b>Mobilesignout</b><br>Test class: test.day3<br>Parameters: secondusername, secondpassword</td>
<td></td>
<td>0</td>
<td>test.day3@3f6f6701</td></tr>
<tr>
<td title='test.day3.Mobilesignout()'><b>Mobilesignout</b><br>Test class: test.day3<br>Parameters: thirdusername, thirdpassword</td>
<td></td>
<td>0</td>
<td>test.day3@3f6f6701</td></tr>
<tr>
<td title='test.day3.WebloginURL()'><b>WebloginURL</b><br>Test class: test.day3<br>Parameters: qaclickacademy.com</td>
<td></td>
<td>0</td>
<td>test.day3@3f6f6701</td></tr>
<tr>
<td title='test.day3.mobileLogin()'><b>mobileLogin</b><br>Test class: test.day3</td>
<td></td>
<td>0</td>
<td>test.day3@3f6f6701</td></tr>
<tr>
<td title='test.day1.secondTest()'><b>secondTest</b><br>Test class: test.day1</td>
<td></td>
<td>0</td>
<td>test.day1@242b836</td></tr>
</table><p>
</body>
</html>